tests/Unit/DataStructures/Test_Variables.cpp

```diff
@@ -318,6 +318,14 @@ void test_variables_math() noexcept {
 
   // Test math +, -, *, /
   const TestVariablesType vars{num_points, value_in_variables};
+  const TestVariablesType close{num_points,
+                                value_in_variables * (1.0 + 1.0e-11)};
+  CHECK(close != vars);
+  CHECK_FALSE(close == vars);
```

Comment:
Add `CHECK_VARIABLES_APPROX(close, vars)` here, or just a comment that the test makes sure that `==` is exact? Else this test is pretty easy to pass (by mistake) by making `close` pretty much anything :D